# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/JSContext.cpp
# Commit: b29a261a06ec
# Full Hash: b29a261a06ec664f02c57121ce6e5302590fa7ac
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-03-04 04:18:28
# Regressor Bug: 1643586
# File Overlap Count: 1
# Description:
#   Bug 1643586 - Part 4: Remove flag for disabling compartment checks from JSContext r=jandem
#   
#   This is only used in one place and it's simpler to check the tracer kind there.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140191
# ==============================================================================

diff -r 619f2aabe4d5 -r b29a261a06ec js/src/vm/JSContext.cpp
--- a/js/src/vm/JSContext.cpp	Thu Mar 03 17:49:17 2022 +0000
+++ b/js/src/vm/JSContext.cpp	Thu Mar 03 17:49:17 2022 +0000
@@ -984,9 +984,6 @@
 #if defined(DEBUG) || defined(JS_OOM_BREAKPOINT)
       runningOOMTest(this, false),
 #endif
-#ifdef DEBUG
-      disableCompartmentCheckTracer(this, false),
-#endif
       inUnsafeRegion(this, 0),
       generationalDisabled(this, 0),
       compactingDisabledCount(this, 0),